spec/system/tasks_spec.rb

```diff
@@ -0,0 +1,102 @@
+require 'rails_helper'
+
+RSpec.describe 'Tasks', type: :system do
+  let(:user) { create(:user) }
+  let(:task) { create(:task, user: user) }
+  let(:another_user) { create(:user, email: 'another_user@example.com', password: 'password', password_confirmation: 'password') }
```

Comment:
パスワードの部分は、定義し直す必要は無さそうですね。
```suggestion
  let(:another_user) { create(:user, email: 'another_user@example.com') }
```

Reply:
2ea7d29 にてFactorybotで定義したものと同じ内容で再定義してしまっていたpassword, password_confirmationを削除しました。